klaviyo/public.py

```diff
@@ -130,3 +128,9 @@ def identify(self, email=None, id=None, properties={}, is_test=False):
 
         query_string = self._build_query_string(params, is_test)
         return self._public_request(self.IDENTIFY, query_string)
+
+    @staticmethod
+    def can_identify(email=None, id=None):
```

Comment:
maybe `def _valid_identifiers(email=None, external_id=None)`

`id` is a reserved word.  Also please add a docstring!

Reply:
static method is cool